### PR TITLE
Test that modulepreload doesn't block window.onload

### PR DESCRIPTION
**Pull request by noamr**

### CODE REVIEW

File: preload/avoid-delaying-onload-link-modulepreload-exec.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/preload/resources/preload_helper.js"></script>
+<link rel=modulepreload href="resources/slow-exec.js">
```
Comment (yoavweiss) [inline preload/avoid-delaying-onload-link-modulepreload-exec.html:5]: I can't find this file. Is it relying on another PR?

File: preload/avoid-delaying-onload-link-modulepreload-exec.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        'HTMLLinkElement.relList is not supported');
+
+    assert_implements(
+        link.relList.supports("modulepreload"),
+        'modulepreload is not supported');
+
+    promise_test(async t => {
+        await new Promise(r => window.addEventListener("load", r));
+
+        assert_false(!!window.didLoadModule);
```
Comment (yoavweiss) [inline preload/avoid-delaying-onload-link-modulepreload-exec.html:19]: This feels racy... I'm concerned that there could be situations where the module load would finish quickly, and then execution would be scheduled before the load event (and delay it, simply by keeping the main thread busy).
Maybe we can guard against that by having slow-exec yielding the main thread once in a while? Not sure what else can defend against flakiness here

Comment (noamr) [reply preload/avoid-delaying-onload-link-modulepreload-exec.html:19]: Yea `slow-exec` has a top level await
Added the missing file

Comment (yoavweiss) [reply preload/avoid-delaying-onload-link-modulepreload-exec.html:19]: Oh, OK. I thought it'd be a busy wait. LG!

**Review by yoavweiss — APPROVED** [review]
LGTM

**Pull request merged**